analytics/analytics-ui/src/app/results/html/html.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit, AfterViewInit } from '@angular/core';
+import { ResultsService } from 'src/app/services/results.service';
+
+@Component({
+  selector: 'app-html',
+  templateUrl: './html.component.html',
+  styleUrls: ['./html.component.css']
+})
+
+export class HtmlComponent implements AfterViewInit {
+
+  constructor(
+    private results: ResultsService,
+  ) { }
+
+  ngAfterViewInit() {
+    const html = this.results.get();
```

Comment:
What happens when there has been no results yet?

Reply:
If you have a look at analytics.service.ts:142 we don't load the results page until the results have loaded. Until then you'll see a buffering spinner symbol once you click execute on the parameters page.